bigquery/samples/tests/conftest.py

```diff
@@ -25,6 +25,22 @@ def client():
     return bigquery.Client()
 
 
+@pytest.fixture
+def project_id(client):
+    return client.project
+
+
+@pytest.fixture
+def table_id(client, dataset_id):
+    now = datetime.datetime.now()
+    table_id = "example_table_{}_{}".format(
+        now.strftime("%Y%m%d%H%M%S"), uuid.uuid4().hex[:8]
+    )
+    table = client.create_table("{}.{}".format(dataset_id, table_id))
+    yield "{}".format(table.full_table_id)
```

Comment:
Don't use the `full_table_id` property for this. Use `table.project`, `table.dataset_id`, and `table.table_id`.

The `full_table_id` property is in legacy SQL format, and also I don't really trust that it's always populated.

Reply:
Updated with your suggestion